### ISSUE TITLE
Object properties "references" and "isReferencedBy" have two separate ranges: "EditorialObject" and "BibliographicalObject"

### ISSUE THREAD

**Issue by JuergenGrupp** [ebu/ebucoreplus#277]
The inverse object properties `references` ad `isReferencedBy` need both of `EditorialObject` and `Bibliographical Object` as range.
The result should allow these statements: 

1. `EditorialObject` `references` only (`BibliographicalObject` or `EditorialObject`)
2. `EditorialObject` `isReferencedBy` only (`BibliographicalObject` or `EditorialObject`)
3. `BibliographicalObject` `references` only (`BibliographicalObject` or `EditorialObject`)
4. `BibliographicalObject` `isReferencedBy` only (`BibliographicalObject` or `EditorialObject`)

**Comment by tormodv** [comment]
Yes, it is a bug, but we can also express the class restrictions as:
_EditorialObject references min 0 BibliographicalObject_ etc which is perhaps what we mean.

**Comment by JuergenGrupp** [comment]
successfully fixed by #393
